src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.scss

```diff
@@ -37,23 +37,51 @@
   pointer-events: none;
 }
 
-.sbb-tooltip-trigger:not([data-icon-small]) {
-  @include sbb.text-inherit;
+.sbb-tooltip-trigger {
+  :host([data-icon-small]) & {
```

Comment:
What if using a tooltip trigger in a form field on a text and not on an icon? I feel that the design is not currently not very clear. There are somehow different kinds of tooltip trigger. Maybe a more general discussion. @kyubisation

Reply:
Should be fine for now. Re-evaluate when we refactor tooltip.